astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -42,24 +41,24 @@ def assert_table_equal(t1, t2, check_meta=False, rtol=1.0e-15, atol=1.0e-300):
     Test equality of all columns in a table, with stricter tolerances for
     float columns than the np.allclose default.
     """
-    assert_equal(len(t1), len(t2))
-    assert_equal(t1.colnames, t2.colnames)
+    npt.assert_equal(len(t1), len(t2))
+    npt.assert_equal(t1.colnames, t2.colnames)
```

Comment:
Here `npt.assert_equal()` is being used to check columns of tables. Compare the tracebacks:
```
============================================== FAILURES ===============================================
__________________________________________ test_example_good __________________________________________

    def test_example_good():
>       assert t1.colnames == t2.colnames
E       AssertionError: assert ['a', 'b', 'c'] == ['a', 'c']
E         At index 1 diff: 'b' != 'c'
E         Left contains one more item: 'c'
E         Use -v to get more diff

example.py:11: AssertionError
__________________________________________ test_example_bad ___________________________________________

    def test_example_bad():
>       np.testing.assert_equal(t1.colnames, t2.colnames)
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: 3
E        DESIRED: 2

example.py:14: AssertionError
```